recipes/spacy-lookups-data/meta.yaml

```
@@ -0,0 +1,49 @@
+{% set name = "spacy-lookups-data" %}
+{% set version = "0.0.4" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/spacy_lookups_data-{{ version }}.tar.gz
+  sha256: 7831260fa75d9a61b3dc16c0853662285e8403ba714042435611277ea9b94ca5
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python
+    - pip
+  run:
+    - python
+
+test:
+  requires:
+    - pytest
```

You also need ``spacy`` here
```suggestion
    - pytest
    - spacy
```

---

Ah true, I'll have to pin that to v2.2.0.

I'll set it to just import the library for now and add the test running later. Otherwise we won't be able to merge this until spaCy v2.2 is out.